test/token/ERC20/ERC20Decrypto.test.js

```diff
@@ -0,0 +1,150 @@
+const { BN, expectEvent, expectRevert, constants } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+const ERC20BurnableMock = artifacts.require('ERC20Decrypto');
+
+contract('ERC20Decrypto', function (accounts) {
+    const [owner, anotherAccount, thirdAccount, ...otherAccounts] = accounts;
+
+    const initialBalance = new BN(1000);
+
+    const name = 'DecryptoToken';
+    const symbol = 'DTKN';
+    const { ZERO_ADDRESS } = constants;
+    const fee = new BN(200);//200 =>2%
+    const zeroBN = new BN(0);
+
+    beforeEach(async function () {
+        this.token = await ERC20BurnableMock.new();
+        await this.token.initialize(name, symbol, owner);
+        await this.token.mint(owner, initialBalance)
+
+    });
+
+    describe('fee', function () {
+        describe('set address fee', function () {
+            it('emits an address fee change event', async function () {
+                const { logs } = await this.token.setAddressFee(anotherAccount, { from: owner });
+                expectEvent.inLogs(logs, 'AddressFeeChange', {
+                    oldOwnerFee: owner,
+                    newOwnerFee: anotherAccount
+                });
+            });
```

Comment:
make _addressFee public, and check if it's the expected one after using the setAddressFee

Reply:
Done!! will be applied in the next PR